README.md

```diff
@@ -820,7 +820,11 @@ Notes:
     - works only for *multi-gpu*.
     - currently only includes the version of *multipack* optimized for linear attention implementations like *flash-attn*.
  * Notes on Fast MoE
-    - `--fast_moe` is an integer value that configures the amount of expert parallel sharding (ep_degree).
+    - `--fast_moe` takes either an integer or boolean value.
+      - When its an integer it enables expert parallel sharding with the given expert parallel degree along with Scatter MoE kernels.
```

Comment:
could you link to Scatter moe kernels

Reply:
I think its too technical and not so useful from a user point of view. Since all the description starts with `fms_acceleration` plugins, I would assume if they are interested to dig deep they would directly visit the fms-acceleration repo.